codec-http2/src/main/java/io/netty/handler/codec/http2/Http2FrameCodec.java

```diff
@@ -582,6 +582,11 @@ void onHttp2FrameStreamException(ChannelHandlerContext ctx, Http2FrameStreamExce
         ctx.fireExceptionCaught(cause);
     }
 
+    final boolean isWritable(DefaultHttp2FrameStream stream) {
+        Http2Stream s = stream.stream;
+        return s != null && connection().remote().flowController().isWritable(s);
```

Comment:
for client side its triggered by this code path:

https://github.com/netty/netty/blob/4.1/codec-http2/src/main/java/io/netty/handler/codec/http2/Http2FrameCodec.java#L397

Reply:
Oh, I see now. I don't feel bad missing that slight-of-hand. That does look like it will break when hitting MAX_CONCURRENT_STREAMS since StreamBufferingEncoder won't create the stream immediately.

So if `s == null` here, then things are weird/broken elsewhere, so it shouldn't really matter whether we consider it writable or not.